pages/code-review.md

```diff
@@ -3,4 +3,125 @@ permalink: /code-review/
 title: Code Review
 ---
 
-# Code Review
```

Comment:
Yessum - the title field is used in the generated template, so it gets duplicated if also present in the body